app/src/pages/organization/organizationProjectsPage/projectsListTable/projectsListTable.jsx

```diff
@@ -0,0 +1,174 @@
+/*!
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useMemo, useState } from 'react';
+import { BubblesLoader, MeatballMenuIcon, Popover, Table } from '@reportportal/ui-kit';
+import classNames from 'classnames/bind';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  activeOrganizationSelector,
+  fetchOrganizationProjectsAction,
+} from 'controllers/organizations/organization';
+import { ProjectName } from 'pages/admin/projectsPage/projectName';
+import { AbsRelTime } from 'components/main/absRelTime';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { loadingSelector, projectsPaginationSelector } from 'controllers/administrate/projects';
+import { messages } from '../messages';
+import styles from './projectsListTable.scss';
+
+const cx = classNames.bind(styles);
+export const ProjectsListTable = ({ projects }) => {
+  const { formatMessage } = useIntl();
+  const [checkedRows, setCheckedRows] = useState(new Set([]));
+  const organizationSlug = useSelector(activeOrganizationSelector).slug;
+  const loadingState = useSelector(loadingSelector);
+  const { order: sortingDirection } = useSelector(projectsPaginationSelector);
```

Comment:
Consider using the `withSortingURL` and `withPagination` Hight order components to connect your page to the URL.
The example from `src/pages/admin/projectsPage/projects/projects.jsx`.
![image](https://github.com/reportportal/service-ui/assets/31861805/219e8856-64ec-4d01-bff6-30afbe1c289b)

The URL params should be taken by saga action to fetch the projects with actual params (sorting & pagination right now, filters as well in the future).